pyccel/parser/semantic.py

```diff
@@ -2310,6 +2311,7 @@ def _visit_CodeBlock(self, expr):
                 if len(line) != 1:
                     errors.report(f"Variable {line[0]} cannot have multiple types",
                             severity='error', symbol=line[0])
+                ls.extend(line)
```

Comment:
This seems to have come from your other branch

Reply:
I suspect that this line is what is causing the unrelated tests to fail

Reply:
Fixed : 0bca7ba